### PR TITLE
initial import of daily goals skill

### PR DESCRIPTION
**Pull request by andrewcatton**

### CODE REVIEW

File: published/Daily_goals_and_results/README [hunk 1]
```diff
@@ -0,0 +1 @@
+Includes actions to ask a user about their goals for the day, follow up later to find out how it went, and to report on the whole team.
```
Comment (attaboy) [inline published/Daily_goals_and_results/README]: Suggested edits:

"Ellipsis will ask members of a team about their goals for the day, follow up later to find out how it went, and report everyone's results to the team."

File: published/Daily_goals_and_results/actions/ask-me-how-my-day-went/triggers.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[ {
+  "text" : "ask me how my day went",
+  "requiresMention" : false,
```
Comment (attaboy) [inline published/Daily_goals_and_results/actions/ask-me-how-my-day-went/triggers.json]: Should we change all these action triggers to requiresMention: true in keeping with our recent discussion?

File: published/Daily_goals_and_results/config.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "name" : "Daily goals & results",
+  "publishedId" : "eZwsrC-zRz-sdgZ0gdrn0Q"
+}
```
Comment (attaboy) [inline published/Daily_goals_and_results/config.json]: Let's add an icon? How about: 🎯

**Comment by andrewcatton** [comment]
Addressed all of your comments

**Pull request merged**